spacemeshos/smapp
# Add PoST generation settings in PoS initialization screen

It closes #1202 

<img width="1442" alt="image" src="https://user-images.githubusercontent.com/1897530/235291773-6e95b80f-f5e0-413e-a83c-5bd90382dd9e.png">

Prerequisite:
- https://github.com/spacemeshos/go-spacemesh/issues/4349

How to test:
1. Clone https://github.com/spacemeshos/post-rs
2. Install `rustc` and `cargo` (if you don't have it yet)
3. `cd ./profiler`
4. Build binaries: `cargo build --release` (it may require running with `sudo`)
5. Copy `post-rs/target/release/profiler` into `smapp/node/mac` directory (or `windows` / `linux`)
6. You also need to have go-spacemesh built from `develop` branch (it requires the latest changes)
7. Set `chmod` and turn off Apple quarantine if needed
8. Connect to the latest devnet
9. Initialize PoS -> You'll see a new screen after selecting PoS directory
